tl/cli/get-phrase-matches.py

```diff
@@ -29,16 +29,34 @@ def add_arguments(parser):
 
     parser.add_argument('input_file', nargs='?', type=argparse.FileType('r'), default=sys.stdin)
 
+    # used for filtering
+    parser.add_argument('--filter', action='store', nargs='?', dest='filter_condition',
+                        default="", help="If set to run filtering, which kind of the data should keep.")
+
 
 def run(**kwargs):
     from tl.candidate_generation import phrase_query_candidates
+    from tl.utility.filter import Filter
```

Comment:
this import should be in the phrase_query_candidates.py in the candidate_generation folder

Reply:
All 3 suggestions fixed